src/EditorFeatures/CSharpTest/SymbolId/SymbolKeyMetadataVsSourceTests.cs

```diff
@@ -70,7 +70,7 @@ public class App : C
             var member02 = (typesym.GetMembers("Prop").Single() as PropertySymbol).Type;
 
             // 'C'
-            var member03 = typesym.BaseType;
+            var member03 = typesym.BaseType();
```

Comment:
Hmm, these tests really shouldn't be using internals like this; can you file a bug against IDE to get this fixed up?

Reply:
I don't think it matters how this test gets to the symbol it needs. It looks like the interesting part is done in ResolveAndVerifySymbol, which operates through public surface.